src/monitorCloudFormationStack.ts

```diff
@@ -0,0 +1,82 @@
+import config from "./config";
+import { DynamoDB } from "aws-sdk";
+import { StackJanitorStatus } from "stackjanitor";
+import { logger } from "./helpers";
+import { DocumentClient } from "aws-sdk/lib/dynamodb/document_client";
+
+const documentClient = new DynamoDB.DocumentClient();
+
+export enum RequestType {
+  CREATE = "CreateStack",
+  UPDATE = "UpdateStack"
+}
+
+export enum Const {
+  EXPIRATION_MESSAGE = "Expiration time should be greater than current time.",
+  SUCCESS = "success",
+  IGNORE = "ignore"
+}
+
+const ExpirationTime: number =
+  new Date().getTime() + 1000 * 60 * 60 * config.DEFAULT_EXPIRATION_HOURS;
+
+const checkExpirationTime = (time: number): boolean =>
+  time > new Date().getTime();
+
+export const putItem = (
+  params: DocumentClient.PutItemInput
+): Promise<DocumentClient.PutItemOutput> => {
+  try {
+    return documentClient.put(params).promise();
```

Comment:
/suggestion Instead of returning the promise, we can probably simplify this to return a structured type that we define, and populate a variable